renderer/components/withIntl.js

```diff
@@ -14,19 +14,11 @@ const getLocale = () => {
   return navigatorLang.split('-')[0]
 }
 
-if (typeof window !== 'undefined' && window.ReactIntlLocaleData) {
-  Object.keys(window.ReactIntlLocaleData).forEach((lang) => {
-    addLocaleData(window.ReactIntlLocaleData[lang])
-  })
-}
-
 if (typeof window !== 'undefined' && window.OONITranslations) {
   supportedMessages = window.OONITranslations
 }
 
 const withIntl = (Page) => {
-  const IntlPage = injectIntl(Page)
```

Comment:
There was no need to call the HoC here. Looks like left over from old template for integrating `react-intl` in `next.js`

Reply:
What's HoC?

Reply:
Higher order Component, it's basically a fancy way of saying that you wrap one component with another one, in this case you are using the `injectIntl` HoC to wrap `Page`.

I believe what this thing was doing is add some instance attributes to the Page to make it work with translations.

I suppose this is no longer needed because in more modern versions of react you can make use of Contexts and therefore it's not needed to wrap stuff in HoCs.